### DISCUSSION TITLE
合并转发消息能够发送到群聊但是不能发送给好友

### DISCUSSION THREAD

**Discussion by VWmin** [mamoe/mirai#966] [Q&A] [answered]
发送到群聊的合并转发，在日志中显示如下：
![image](https://user-images.githubusercontent.com/34272476/106849058-06a9b680-66ed-11eb-9f67-cc593e65a7d2.png)

发送给好友的合并转发，在日志中显示如下：
![image](https://user-images.githubusercontent.com/34272476/106849525-cd257b00-66ed-11eb-893e-9f3c9710e9a7.png)

构建转发消息的方式
```java
 ForwardMessageBuilder builder = new ForwardMessageBuilder(group);
 builder.says(bot, "hello")
         .says(bot, "word");
 target.sendMessage(builder.build());
```

版本：2.0-M2-2

**Comment by Karlatemp** [comment, marked as answer]
版本过旧

**Reply by VWmin** [reply]
我麻了...问题解决，感谢
